content/applications/general/iot/iot_box.rst

```diff
@@ -0,0 +1,91 @@
+=======
+IoT box
+=======
+
+To start using an IoT box:
+
+#. Make sure you have a valid IoT box subscription in addition to your Odoo subscription.
+#. Connect your :doc:`devices <devices>` to the IoT box.
+#. :ref:`Connect the IoT box <iot/iot_box/network>` to the network.
+#. :doc:`Connect the IoT box to your Odoo database <connect>`.
+
+.. note::
+   Devices can also be connected after the IoT box is added to the network and/or connected to the
+   database; however, a reboot of the IoT box is required.
+
+.. seealso::
+   `IoT system FAQ <https://www.odoo.com/app/iot-faq>`_
+
+.. _iot/iot_box/network:
+
+Network connection
+==================
+
+The IoT box can be connected to the network via :ref:`Ethernet <iot/iot_box/network-ethernet>` or
+:ref:`Wi-Fi <iot/iot_box/network-wifi>`.
+
+.. important::
+   The IoT box must be connected to the same network as the devices connected to Odoo and the ones
+   connected to the IoT box.
+
+.. _iot/iot_box/network-ethernet:
+
+Ethernet
+--------
+
+Connect the IoT box to a power source, then plug the Ethernet cable into the IoT box's Ethernet port
+and an available port on your router.
```

Comment:
technically speacking it can not be directly on the router to work. Maybe we can add a little note with somethin like "if you have a technical network administrator, you can check with them. The IoT should be in the local area network (LAN) with the IoT devices and the devices using Odoo IoT features"